Client/src/Components/TerryComponent/index.jsx

```diff
@@ -1,7 +1,95 @@
-import React from "react";
+// eslint-disable-next-line no-unused-vars
+import React, { useState } from "react";
+import { Button, Container } from "@mui/material";
+import AlertMessage from "./reusable/AlertMessage";
 
-const index = () => {
-  return <div></div>;
+const Index = () => {
+  const [notification, setNotification] = useState({
+    open: false,
+    title: "",
+    message: "",
+    type: "",
+    alertMessageType: "",
+    actionText: "",
+  });
+
+  const handleOpenAlertMessage = (type, title, message, actionText) => {
```

Comment:
Given the number of parameters here I might consider passing this function a config object instead:
```
const handleOpenAlertMessage = (config) => {
  ...
}
```
then you can simply pass it all your params:

```
{
  type: "type",
  title: "title",
  message: "message",
  actionText: "actionText
}
```

The JSdocs would then describe the shape of the config object.

This is not a hard and fast requirement, but it's something to think about when your function start taking many parameters.

At the very least the function requires JSdocs so that we know what parameters to give it.